projects/managed-hubs/incidents.md

```diff
@@ -168,6 +178,7 @@ This lets us use notes, status updates from pagerduty as well as messages from S
 5. **Add Data Sources** that we will use to keep track of the actions that happened around the incident.
    - Link to the slack channel we created for this incident as a "Data Source", filled in with an appropriate time to cover all the messages there.
    - Add any other channels where there was conversation there about the incident (e.g., GitHub Issues or Pull Requests).
+     % How can you add GitHub data sources?
```

Comment:
The docs say we should add GitHub issues and PRs as data sources to the incident report, but not _how_. I couldn't figure this out. The postmortem only offered me PagerDuty and "Connect to Slack" which then let me select the slack channel. There was no similar "Connect to GitHub" or even place to paste a normal URL.

How do we actually do this?

Reply:
@sgibson91 in https://2i2c-org.pagerduty.com/incidents/Q11W35VTVPW6BD, I see this:

![image](https://user-images.githubusercontent.com/30430/204423176-d22fdcbe-fb76-4335-8be2-2a52112455c0.png)

This is GitHub PRs made during the time of the outage i think. I'm not exactly sure how to add these to the postmortem though :|

Reply:
Right, I couldn't find anything illuminating in the PagerDuty docs either. I've completed the LIS report without these. I think moving forward we should either:

1. Figure out how to add GitHub events to the postmortem, or
2. Remove this instruction because it's confusing and unclear.

Reply:
Maybe remove the instruction but add a todo comment to add instructions about how to add GitHub events once we know?

Reply:
@GeorgianaElena suggestion seems reasonable to me.

Reply:
Done [`aad4a8e` (#571)](https://github.com/2i2c-org/team-compass/pull/571/commits/aad4a8e0d067612b15b956327c2e230a110e5c8c)